types/mydecimal.go

```diff
@@ -250,6 +251,39 @@ type MyDecimal struct {
 	wordBuf [maxWordBufLen]int32
 }
 
+// SerializeForSpill serializes MyDecimal to bytes
+func (d *MyDecimal) SerializeForSpill(buf []byte) (int64, error) {
+	// TODO
+	return -1, nil
+}
+
+// DeserializeForSpill deserializes MyDecimal to bytes
+func (d *MyDecimal) DeserializeForSpill(buf []byte) (int64, error) {
```

Comment:
This is not needed. What we really want to do is to append a decimal into a chunk.Column.
We can use chunk.AppendDecimal directly. It already decodes a decimal to a []byte.